chef_master/source/release_notes_chefdk.rst

```diff
@@ -1,10 +1,65 @@
 =====================================================
-Release Notes: Chef Development Kit 0.19 - 3.3
+Release Notes: Chef Development Kit 0.19 - 3.4
 =====================================================
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/release_notes_chefdk.rst>`__
 
 Chef Development Kit is released on a monthly schedule with new releases the third Monday of every month. Below are the major changes for each release. For a detailed list of changes see the `ChefDK Changelog on GitHub <https://github.com/chef/chef-dk/blob/master/CHANGELOG.md>`__
 
+What's New in 3.4
+=====================================================
+
+* **Chef 14.6.47**
+
+  ChefDK now ships with Chef 14.6.47. See `Chef 14.6 release notes </release_notes.html#whats-new-in-14-6>`__ for more information on what's new.
+
+* **Smaller package size**
+
+  ChefDK RPM and Debian packages are now compressed. Additionally, many gems were updated to remove extraneous files that do not need to be included. The download size of packages has decreased accordingly (all measurements in megabytes):
+
+    * .deb: 108 -> 84 (22%)
+    * .rpm: 112 -> 86 (24%)
+
+* **Platform Additions**
+
+  macOS 10.14 (Mojave) is now fully tested and packages are available on the `downloads <https://downloads.chef.io/chefdk/3.4.38>`__ page.
+
+* **Updated Tooling**
+
+  **Fauxhai**
+
+    Foodcritic has been updated from 14.1 to 14.3. This updates the metadata that ships with Foodcritic to provide the latest Chef 13.11 and 14.5 metadata, while removing metadata from older Chef releases. This update also removes the FC121 rule, which was causing confusion with community cookbook authors. This rule will be added back when Chef 13 goes EOL in April 2019.
```

Comment:
> oodcritic has been updated from 14.1 to 14.3. This updates the metadata that ships with Foodcritic to provide the latest Chef 13.11 and 14.5 metadata, while removing metadata from older Chef releases

Suggest: Foodcritic has been updated from 14.1 to 14.3. This brings its metadata up to date with Chef 13.11 and 14.5 and removes older Chef metadata.